src/components/HomePage/UpcomingEvents.js

```diff
@@ -29,29 +39,45 @@ function UpcomingEvents() {
 	const currentTime = new Date();
 	let eventsDisplayed = 0;
 	const numEventsToDisplay = 2;
-	const timeSlots = [];
+
+	const events = [];
 	// Assumes workshopSchedule is ordered by time
 	for (let i = 0; i < workshopSchedule.length && eventsDisplayed < numEventsToDisplay; i++) {
-		const workshop = workshopSchedule[i];
-		if (currentTime < workshop.startTime) {
-			timeSlots.push(workshop);
-			eventsDisplayed += workshop.events.length;
+		const timeSlot = workshopSchedule[i];
+		if (currentTime < timeSlot.startTime) {
+			timeSlot.events.forEach(event => {
+				const eventFormat = {
+					startTime: `${timeFormatter.format(timeSlot.startTime)} ${currentTimeZoneShort}`,
+					...event
+				};
+				events.push(eventFormat);
+			});
```

Comment:
Don't use `forEach`. Use a `for`-`of` loop instead.
```suggestion
			for (const event of timeSlot.events) {
				events.push({
					startTime: `${timeFormatter.format(timeSlot.startTime)} ${currentTimeZoneShort}`,
					...event
				});
			}
```